### DISCUSSION TITLE
Option to disable updating /back teleport location when passing through portals

### DISCUSSION THREAD

**Discussion by siulung201314** [Zrips/CMI#6118] [Feature Requests]
When players are passing through a portal such as nether portal, end portal and end gateway, the last position is saved near the portal.
Players will be teleported near the portal when using back command.
It would be nice to have an option to disable saving the location when passing through those portals. Therefore, when players pass through the portal, it would not update the player's last teleport location.

**Comment by wling-art** [comment]
Yes, I need this feature too, a lot of players are complaining about this.
